common/clap_utils/src/lib.rs

```diff
@@ -190,3 +192,11 @@ where
     }
     Ok(())
 }
+
+pub fn get_color_style() -> Styles {
+    Styles::styled()
+        .header(AnsiColor::Yellow.on_default())
+        .usage(AnsiColor::Green.on_default())
+        .literal(AnsiColor::Green.on_default())
+        .placeholder(AnsiColor::Green.on_default())
```

Comment:
Will this render all text as green?

Reply:
no, it looks like this:

![image](https://github.com/sigp/lighthouse/assets/7217615/4573e523-8d33-4043-8952-f1d46b10c508)